runtime/runtime/src/lib.rs

```diff
@@ -3465,19 +3473,12 @@ pub mod estimator {
         // For the estimator, create a limitless receipt sink that always
         // forwards. This captures congestion accounting overhead but does not
         // create unexpected congestion in estimations.
```

Comment:
Not related to your PR but this comment no longer matches the code. In order to make it limitless we should use the relevant runtime config to disable cc.

Reply:
Right. Let me remove the outdated comment and put another TODO there... Their number seems to go towards zero but not monotonically :sweat_smile: